torch/csrc/jit/ir.cpp

```diff
@@ -245,7 +245,8 @@ std::ostream& printNode(std::ostream & out, Node * n, std::vector<Node*> * group
   } else {
     emitUses(out,n);
   }
-  out << "];\n";
+  out << "], ";
+  out << "scope: " << n->scopeName() << ";\n";
```

Comment:
can you make printing of the scope optional based on whether it is defined? It's unnecessary if it was not used and this way we won't have to change all expect files as well :)

Reply:
Done